service/src/test/java/greencity/service/EcoNewsServiceImplTest.java

```diff
@@ -107,8 +107,8 @@ void save() throws MalformedURLException {
     @Test
     void saveWithExistedImage() throws IOException {
```

Comment:
I think that test case isn't relevant anymore, so you can comment/remove it fully.